README.md

```diff
@@ -26,14 +26,15 @@ Furthermore, NetworKit's core can be built and used as a native library if neede
 You will need the following software to install NetworKit as a python
 package:
 
-- A modern C++ compiler, e.g.: [g++] (&gt;= 5.3) or [clang++] (&gt;= 3.9)
+- A modern C++ compiler, e.g.: [g++] (&gt;= 5.3), [clang++] (&gt;= 3.9) or MSVC (&gt;= 14.13)
 - OpenMP for parallelism (usually ships with the compiler)
 - Python3 (3.5 or higher is supported)
   - Development libraries for Python3. The package name depends on your distribution. Examples: 
     - Debian/Ubuntu: `apt-get install python3-dev`
     - RHEL/CentOS: `dnf install python3-devel`
+    - Windows: Use the official release installer from [www.python.org][https://www.python.org/downloads/windows/]
 - [Pip]
-- [CMake] version 3.5 or higher (e.g., `pip3 install cmake`)
+- [CMake] version 3.16 or higher (e.g., `pip3 install cmake`)
```

Comment:
Are there any distributions that do not have 3.16 available? 3.16 is only 2 years old, is it?.

Reply:
Thats certainly possible. Thats why we suggest to install cmake via `pip3`. Since they deploy via wheel-package, it is supported up to CentOS 5.

Another way would be to document, not to mix conda and pip environments.

Reply:
Based on external discussion: Setting the minimum cmake version to 3.6 (youngest release in 5 year support span). Also adding an conditional for `FindPython` (supported since 3.12). For older versions we use `FindPythonInterp` and `FindPythonLibs`.